<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceManager</class>
 <widget class="QMainWindow" name="DeviceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linien - Spectroscopy lock - Device manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>510</width>
     <height>280</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="deviceList"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>New device</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove device</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>DeviceManager</receiver>
   <slot>connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>DeviceManager</receiver>
   <slot>new_device()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>DeviceManager</receiver>
   <slot>remove_device()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deviceList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>DeviceManager</receiver>
   <slot>selected_device_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>remove_device()</slot>
  <slot>selected_device_changed()</slot>
  <slot>connect()</slot>
  <slot>new_device()</slot>
 </slots>
</ui>
